### ISSUE TITLE
Setting sparklineGroups.MaxAxisType to eSparklineAxisMinMax.Group does not work

### ISSUE THREAD

**Issue by Chewieez** [EPPlusSoftware/EPPlus#864]
I am using EPPlus 6.2.2 in .Net 6 and am seeing that when I set the `sparklineGroups.MaxAxisType = eSparklineAxisMinMax.Group`, that setting is not maintained when the excel file is opened in Excel. The setting is always set to "Automatic for each sparkline" or `eSparklineAxisMinMax.Individual`

I can recreate it with a simple worksheet as follows:

```
using var excel = new ExcelPackage();
            var worksheet = excel.Workbook.Worksheets.Add("Test");

            worksheet.Cells[1, 1].Value = 1100;
            worksheet.Cells[1, 2].Value = 2200;
            worksheet.Cells[1, 3].Value = 3300;
            worksheet.Cells[1, 4].Value = 4400;
            worksheet.Cells[2, 1].Value = 2000;
            worksheet.Cells[2, 2].Value = 1254;
            worksheet.Cells[2, 3].Value = 5423;
            worksheet.Cells[2, 4].Value = 1400;
            worksheet.Cells[3, 1].Value = 2343;
            worksheet.Cells[3, 2].Value = 2355;
            worksheet.Cells[3, 3].Value = 2121;
            worksheet.Cells[3, 4].Value = 1231;
            worksheet.Cells[4, 1].Value = 954;
            worksheet.Cells[4, 2].Value = 4323;
            worksheet.Cells[4, 3].Value = 1112;
            worksheet.Cells[4, 4].Value = 2211;

            var sparklineGroups = worksheet.SparklineGroups.Add(
                        eSparklineType.Line,
                        worksheet.Cells[1, 6, 4, 6],
                        worksheet.Cells[1, 1, 4, 4]
                    );

            sparklineGroups.MinAxisType = eSparklineAxisMinMax.Custom;
            sparklineGroups.ManualMin = 0;
            sparklineGroups.MaxAxisType = eSparklineAxisMinMax.Group;

            using var outputStream = new MemoryStream();

            excel.SaveAs(outputStream);
```

#
![EPPlus_Sparkline_Group_MaxAxisType](https://user-images.githubusercontent.com/18487182/235543019-37f7124e-8328-4aa4-bca5-26df6df1cb52.png)

Am I doing something wrong or is this a bug? 

Thanks!

**Comment by JanKallman** [comment]
Thanks for reporting this. It's a bug. I will provide a fix shortly.

**Comment by JanKallman** [comment]
Fixed in EPPlus 6.2.3
